group_vars/irods.yml

```diff
@@ -7,28 +7,30 @@ firewall_allowed_tcp_ports:                 # list of open ports on iCAT server
   - "5432"  # PostgreSQL
   - "20000:20199"  # irods
 
-ir_version: '-4.2.11*'                  # if defined (empty): version will be installed (must start with '-' and end with '*')
+ir_version: '-4.3.0*'                  # if defined (empty): version will be installed (must start with '-' and end with '*')
```

Comment:
Do we need the `*` at the end? If we really need it, then I think we should first add a character to signal the end of the version number before the `*` to prevent surprises:
`-4.3.1*` will match both `-4.3.1-*` as well as `-4.3.11-*`.
How about simply
```
ir_version: 4.3.0
```
and in [roles/irods/tasks/icat_pre.yml](https://github.com/rug-cit-hpc/league-of-robots/blob/0fbf588f8ac167ec0bd7a331af92554e0dbd286d/roles/irods/tasks/icat_pre.yml):
```
- name: Installing the iRODS Package(s)
  ansible.builtin.yum:
    name: '{{ item }}'
    state: present
  loop:
    - irods-server-{{ ir_version }}-*
    - irods-database-plugin-postgres-{{ ir_version }}-*
    - irods-runtime-{{ ir_version }}-*
    - irods-resource-plugin-s3-{{ ir_version }}-*
    - irods-rule-engine-plugin-python-{{ ir_version }}-*
    - irods-rule-engine-plugin-unified-storage-tiering-{{ ir_version }}-*
  become: true
```

Reply:
Fixed. Made two variables - both needs to be defined (in defaults set 4.3.0/4.3.0.0): one for irods and and core packages, second one for plugins.